### PR TITLE
Doc/overall edit

### PR DESCRIPTION
**Pull request by PipKat**
I'm submitting this PR as a draft because I'm sure there will be several issues to resolve.

### CODE REVIEW

**Comment by PipKat** [comment]
@chrpetre I'm not as proficient with resolving testing issues as I'd like to me. I thought I updated the pytests to use my new message strings, but all the checks that are failing are pytest related. Would you or a tester on your team be able to update the tests and possibly even pass on any tips so that I can do a better job at resolving them myself? Thanks!

File: doc/source/api/index.rst [hunk 1]
```diff
@@ -4,33 +4,34 @@
 API reference
 =============
 
-This section describes the core Pythonic interfaces for Twin Runtimes.
-Here, you can find the application programming interfaces for 
-using the core Twin Runtime SDK functionalities, as well as an example
-of higher level abstraction class to evaluate the Runtimes.
+This section describes the core Pythonic interfaces for twin runtimes.
+Here, you can find the APIs for using twin runtime functionalities
```
Comment (PipKat) [inline doc/source/api/index.rst:8]:
```suggestion
Here, you can find the APIs for using twin runtime functionalities and
```

[the remark this answers is not in the document]
Comment (PipKat) [reply doc/source/api/index.rst:8]: @chrpetre Thanks! It was missing the "and" and the end of line 8. Will someone on the team be able to address the pytest errors?

[the remark this answers is not in the document]
Comment (chrpetre) [reply doc/source/api/index.rst]: thanks @PipKat , @lboucin is going to look at the pytest errors later on today normally

[the remark this answers is not in the document]
Comment (PipKat) [reply doc/source/api/index.rst]: @lboucin If you have tips on how I should have/could have resolved the errors myself, I'd love to hear them! I know I caused them all by editing message strings. Thanks!

**Pull request closed without merging**